### PR TITLE
Update nodejs-6 to 6.8.1

### PR DESCRIPTION
**Pull request by AndWac**

### CODE REVIEW

File: components/runtime/node6/nodejs.p5m [hunk 1]
```diff
@@ -13,15 +13,14 @@
...
+
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
 set name=pkg.summary value="$(COMPONENT_SUMMARY)"
 set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-
 license $(COMPONENT_LICENSE_FILE) license="$(COMPONENT_LICENSE)"
-
```
Comment (pyhalov) [inline components/runtime/node6/nodejs.p5m:24]: manifest seems to get some unnecessary whitespace changes. Please, fix manifest header fields.

File: components/runtime/node6/nodejs.p5m [hunk 2]
```diff
@@ -2783,10 +2902,12 @@ file path=usr/node/6/lib/node_modules/npm/scripts/doc-build.sh
...
 file path=usr/node/6/lib/node_modules/npm/scripts/install.sh
+file path=usr/node/6/lib/node_modules/npm/scripts/maketest
 file path=usr/node/6/lib/node_modules/npm/scripts/publish-tag.js
 file path=usr/node/6/lib/node_modules/npm/scripts/release.sh
 file path=usr/node/6/lib/node_modules/npm/scripts/relocate.sh
 file path=usr/node/6/lib/node_modules/npm/scripts/update-authors.sh
 file path=usr/node/6/share/doc/node/gdbinit
 file path=usr/node/6/share/man/man1/node.1
 file path=usr/node/6/share/systemtap/tapset/node.stp
+
```
Comment (pyhalov) [inline components/runtime/node6/nodejs.p5m]: empty line

**Comment by AndWac** [comment]
Hi,
I hope I understood your comments properly.

**Comment by pyhalov** [comment]
Seems so. Why did you close PR?

**Comment by AndWac** [comment]
Sorry, that wasn't intentionally. Must have hit the wrong button...
